modules/ROOT/pages/event-tracking.adoc

```diff
@@ -109,20 +110,20 @@ In `mule-agent.yml`:
 [source,yaml,linenums]
 ----
 services:
-  mule.agent.tracking.service:
-    globalTrackingLevel: TRACKING
+ mule.agent.tracking.service:
+ globalTrackingLevel: TRACKING
```

Comment:
seems here the 2 spaces or 1 tab are required here since its a yaml file

Reply:
yaml validator says this is valid. And the bug that was filed said the space wasn't needed. So...